README.md

```diff
@@ -30,16 +30,53 @@ month: 2
 year: 2023
 ---
 
-your markdown content
+# This is a heading style one
+
+**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
+
+More text
+
+::image[d394zg4utwqxynenamqf]
+
+## This is a heading in heading style two
+
+This is a standard paragraph with a [LINK](https://www.google.com). **This is bold paragraph text**
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can see how that looks at differnt sizes adn make sure it looks as it is intended too
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it. 
+
+### This is a heading in heading style three introducing a video
+
+https://www.youtube.com/watch?v=dB_3R84ewig
```

Comment:
Is this intended to be an embedded video or just a link? I'm not confident this will render as a clickable link (and I can't check as we do not have dev docs yet 😕).

Reply:
@natestemen yeah it's strange but it should render as an embedded frame. I just tested it [here](https://unitary-fund-prel-lj20ttudw-unitary-fund.vercel.app/posts/test) in [this PR](https://github.com/unitaryfund/unitary.fund/pull/367). I believe it is a special feature implemented by the designers, not a general property of md. 

I think the same applies for paragraph formatting, that is why I suggest keeping reference to it.